Sources/Starknet/Providers/StarknetProvider/JsonRpcParams.swift

```diff
@@ -62,3 +62,5 @@ struct GetClassHashAtParams: Encodable {
         case blockId = "block_id"
     }
 }
+
+struct GetBlockNumberParams: Encodable {}
```

Comment:
Let's get rid of it.

Let's make `params` in `JsonRpcPayload` optional and argument `params` in `makeRequest` inside `StarknetProvider` as well.